tests/build/tkn-bundle.go

```diff
@@ -31,7 +31,7 @@ import (
   - kubectl patch serviceaccount appstudio-pipeline -p '{"imagePullSecrets": [{"name": "docker-config"}], "secrets": [{"name": "docker-config"}]}'
 */
 
-var _ = framework.TknBundleSuiteDescribe("tkn bundle task", Label("tasks", "HACBS"), func() {
+var _ = framework.TknBundleSuiteDescribe("tkn bundle task", Label("build-service", "tasks"), func() {
```

Comment:
these tests are not directly related to `build-service`, I think it was planned to run in [build-definitions repo](https://github.com/konflux-ci/build-definitions) CI changes, will it be good to change `build-service` to `build-templates`?